src/main/java/seedu/address/model/task/Deadline.java

```diff
@@ -0,0 +1,4 @@
+package seedu.address.model.task;
+
+public class Deadline {
```

Comment:
Is it possible to add a constructor with some parameters instead of just an empty class?